tests/unit/common/core/test_main.py

```
@@ -252,3 +252,45 @@ def test_ensure_cli_env__env_service_name__expected_otel_service_name(
         endpoint="http://collector:4318/v1/traces",
         service_name="my-custom",
     )
+
+
+@pytest.mark.parametrize(
+    "subcommand,handler_name",
```

Ruff (0.15.20)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)